src/game.py

```diff
@@ -27,24 +27,21 @@ def get_prompt(self, total_marks_on_board):
         else:
             message.display_prompt_message_for_move(self, current_player)
 
-    def get_prompt_for_occupied_spot(self, board, user_input):
-        if self.new_board.is_spot_taken(board, user_input):
-            message.display_spot_taken_message(self)
+    def get_prompt_for_occupied_spot(self):
+        message.display_spot_taken_message(self)
 
     def process_user_input(self):
         user_input_as_string = self.get_user_input()
         position_choice = self.convert_input_to_integer(user_input_as_string)
-        # look at - not working - or should it go somewhere else?
         if self.new_board.is_spot_taken(self.board, position_choice):
-            self.get_prompt_for_occupied_spot(self.board, position_choice)
-        else:
-            self.new_board.mark_board(
-                position_choice,
-                self.board,
-                self.get_current_player(self.total_marks_on_board),
-            )
-            self.total_marks_on_board = self.new_board.count_marks(self.board)
-        # do I need this return?
+            self.get_prompt_for_occupied_spot()
+            self.process_user_input()
+        self.new_board.mark_board(
```

Comment:
outside the scope of this PR, but I’d love to see a rename on the class variable new_board, since I would think a “new board” would be empty 100% of the time. Maybe game_board?

Reply:
will put a small card on the board to get this done.